test/langtools/jdk/javadoc/doclet/testMissingComment/TestMissingComment.java

```diff
@@ -112,6 +112,7 @@ private void test(Path base, String code, String expect) throws Exception {
 
         javadoc("-d", base.resolve("api").toString(),
                 "-Xdoclint:missing",
+                "--no-platform-links",
```

Comment:
More of those... sigh.

Reply:
A different solution would be to disable platform links by default (in JavadocTester) and only enables for specific tests.  But that would be a minor paradigm shift of JavadocTester providing default options.